SAP/luigi
# POC E2E with fiddle configurations

A proof of concept to test our e2e tests against luigi fiddle with smaller custom configurations.

The POC runs against the production version of fiddle, but could also be extended to install and spin up a local instance, so the current version of Luigi can be used. This is required for new features.

Additionally updated to release-v1 and fiori-fundamentals.

How to test:
Setup: `lerna bootstrap --force-local && lerna run bundle && lerna run build`
Then run the e2e tests: `./test/e2e.sh`   > you should see 0-fiddle/fiddle-navigation.spec.js
